dev/test/gapic-firestore-v1.ts

```diff
@@ -11,76 +11,134 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
 
-'use strict';
```

Comment:
It would be interesting to know if the old system tests pass against the new client. Did you test that by any chance?